internal/router/router.go

```
@@ -50,3 +53,47 @@ type LocalRouter interface {
+			return &p, p.Models[0], true
+		}
+		return &p, "", true
+	}
+	return nil, "", false
+}
+
+// resolvePeerBaseModel checks whether search is a virtual sub-ID (a key in any
+// peer's SetParamsByID) and returns the peer's base model name. Peers are
+// visited in sorted order (first-wins for duplicates). Returns the raw search
+// string when no match is found.
+func resolvePeerBaseModel(cfg config.Config, search string) string {
```

golangci-lint (2.12.2)

[error] func resolvePeerBaseModel is unused

(unused)